.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macOS-latest, windows-latest ]
+        python-version: [ '3.8' ]
```

Comment:
Is there a reason why we want to avoid `python-version: ['3.8', '3.9', '3.10']` so the tests runs on all three different of python for also `mac` and `windows`?

Reply:
The main reason is to limit the number of used runners as you might end up in a queue if you use too many runners and you have to wait until all are finished.
Hence I decided to separate testing Python versions and operating systems. As Linux is the most important OS to us, we test Python versions on Linux. And as Python 3.8 is our current standard, we test operating systems for that version.

Reply:
I see limiting the number of used runners makes sense then.